widget/embedded/src/store/slices/data.ts

```diff
@@ -68,6 +79,14 @@ export const createDataSlice: StateCreator<
       return blockchainsFromState;
     }
 
+    if (options.type === 'custom') {
+      return blockchainsFromState.filter(
+        (blockchain) =>
+          blockchain.type === TransactionType.EVM ||
```

Comment:
Please create a const called `ACTIVE_BLOCKCHAINS_FOR_CUSTOM_TOKENS` and put it outside of this method to clearly communicate the logic and also simple add more blockchains in future.